testdata/p4_16_errors/structured-annotation-e2-ConstantFolding_0_DoConstantFolding.p4

```diff
@@ -0,0 +1,5 @@
+@badType[a={ 2, 3 }] control c() {
```

Comment:
Also no expected output for this test program, nor the one named structured-annotation-e2-FrontEnd_5_ConstantFolding.p4

Reply:
This one shouldn't be here.